src/components/TimecodeTracker/TimecodeTracker.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import {
+  getTimecodeGameLength,
+  getTimecodeTotalLength,
+} from '../../utils/Timecode'
+import TimecodeInput from './TimecodeInput'
+import TimecodeResult from './TimecodeResult'
+
+export default function TimecodeTracker() {
+  const [timecodes, setTimecodes] = useState(['', '', ''])
+
+  return (
+    <>
+      <form onKeyDown={handleEnter}>
+        {timecodes.map((_, index) => (
+          <TimecodeInput
+            title={index < 2 ? 'Halbzeit ' + (index + 1) : 'O-Töne'}
+            index={index}
+            key={index}
+            onChange={handleChange}
+          />
+        ))}
+      </form>
+      <TimecodeResult
+        text="Länge Spiel"
+        result={getTimecodeGameLength(timecodes)}
+      />
+      <TimecodeResult
+        text="Länge gesamt"
+        result={getTimecodeTotalLength(timecodes)}
+      />
+    </>
+  )
+
+  function handleChange(event, index) {
+    const value = event.target.value
+    const oldTimecodes = [...timecodes]
+    oldTimecodes[index] = value
+    setTimecodes(oldTimecodes)
+  }
+
+  function handleEnter(event) {
+    if (event.keyCode === 13) {
```

Comment:
```suggestion
    if (event.keyCode === enter) {
```

Reply:
No, keyCode is a specific code linked to a key
but i can use event.key = "Enter" instead. will think about that.

Reply:
use event.key = "Enter" is what I meant. I think its better because you don't have to look up what keyCode 13 is.